src/win/util.c

```diff
@@ -1663,6 +1673,127 @@ int uv_os_unsetenv(const char* name) {
 }
 
 
+static int uv__gethostnamew_nt60(WCHAR* name, int name_len) {
+  int result_len;
+  int error_code = NO_ERROR;
+
+  /* WSALookupService stuff
+   * Avoid dynamic memory allocation if possible */
+  CHAR local_buf[WSAQ_LOCAL_BUF_LEN];
+  DWORD dwlen = (DWORD)WSAQ_LOCAL_BUF_LEN;
```

Comment:
These are still illegal syntax expressions (for C89)

Reply:
What compiler are you using?
-std=c89 -Wall -Wextra -pedantic compiles fine with MinGW.
How is this different from this one?
https://github.com/libuv/libuv/blob/v1.x/src/win/tcp.c#L1597

Reply:
ah, my mistake. I'm using the wetware compiler, but

Reply:
constant integer expressions are also allowable (https://en.cppreference.com/w/c/language/array#Arrays_of_constant_known_size)